openbb_platform/core/openbb_core/api/rest_api.py

```diff
@@ -85,6 +85,16 @@ async def lifespan(_: FastAPI):
 
 
 if __name__ == "__main__":
+    # pylint: disable=import-outside-toplevel
     import uvicorn
 
-    uvicorn.run("openbb_core.api.rest_api:app", reload=True)
+    Env()
```

Comment:
Is this a leftover from a previous implementation?

Reply:
if it is - it's also mentioned in the `openbb_platform/extensions/platform_api/openbb_platform_api/main.py`

Reply:
This is to bring the environment variables into the context of the script entry point, so variables can be read and established before launching. Otherwise they are not yet exposed when invoking uvicorn.

Reply:
Ok, I suggest we leave a comment saying that we are loading the .env such that `uvicorn` has access to the environment variables x, y, z.